singularity_test.sh

```diff
@@ -43,5 +46,5 @@ fi
 
 # See https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
 this_script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-simg=$this_script_dir/alpine.simg
-timeout $timeout_length singularity exec $simg ls $@
+simg="${this_script_dir}/alpine.simg"
+timeout "$timeout_length" singularity exec "$simg" ls "$@" > /dev/null 2>&1
```

Comment:
How about redirect stdout but not stderr, so we get to see the list of things that `ls` didn't find?

Reply:
Yeah, good point!